pkg/auth/strategy.go

```diff
@@ -29,7 +29,8 @@ type Options struct {
 	TokenEndpoint         string `mapstructure:"token_endpoint"`
 	IssuerURL             string `mapstructure:"issuer_url"`
 
-	SSLDisabled bool `mapstructure:"ssl_disabled"`
+	UseBasicAuth bool `mapstructure:"use_basic_auth"`
```

Comment:
Rename this to `IsBrokenAuthHeader`?

Reply:
`UseBasicAuth` is more explicit about what is done and it's more concise

Reply:
`OAuthWithBasic`? UseBasicAuth on an `auth.Options` looks like you're using basic authentication and not OAuth

Reply:
Renamed it to `TokenBasicAuth` so it is clear that the token is fetched via basic auth
This is also consistent with the name used in service-manager